net/advanced-operations/printing/print-dotnetcore/_index.md

```diff
@@ -85,102 +85,215 @@ The Aspose.PDF library allows us to convert PDF files to XPS. This function can
+{{< tab tabNum="1" >}}
+```csharp
+// For complete examples and data files, visit https://github.com/aspose-pdf/Aspose.PDF-for-.NET
 
-```cs
-using (Document document = new Document())
+private static void PickTrayByPdfSize()
 {
-    Page page = document.Pages.Add();
-    page.Paragraphs.Add(new TextFragment("Hello world!"));
+    // The path to the documents directory
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create a new document
+    using (var document = new Aspose.Pdf.Document())
+    {
+        // Add a new page and some content
+        Aspose.Pdf.Page page = document.Pages.Add();
+        page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Hello world!"));
+
+        // Set the flag to choose a paper tray using the PDF page size
+        document.PickTrayByPdfSize = true;
+
+        // Save the document
+        document.Save(dataDir + "PickTrayByPdfSize_out.pdf");
+    }
+}
+```
+{{< /tab >}}
+
+{{< tab tabNum="2" >}}
+```csharp
+// For complete examples and data files, visit https://github.com/aspose-pdf/Aspose.PDF-for-.NET
+
+private static void PickTrayByPdfSize()
+{
+    // The path to the documents directory
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create a new document
+    using var document = new Aspose.Pdf.Document();
+
+    // Add a new page and some content
+    Aspose.Pdf.Page page = document.Pages.Add();
+    page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Hello world!"));
 
     // Set the flag to choose a paper tray using the PDF page size
     document.PickTrayByPdfSize = true;
-    document.Save("result.pdf");
+
+    // Save the document
+    document.Save(dataDir + "PickTrayByPdfSize_out.pdf");
 }
 ```
+{{< /tab >}}
+{{< /tabs >}}
 
 ## Print Dialog Presets Page Scaling
 
 The next code snippet is intended to ensure that the [PrintScaling](https://reference.aspose.com/pdf/net/aspose.pdf/document/printscaling/) property is correctly applied and saved in the PDF.
 
-The [PrintScaling](https://reference.aspose.com/pdf/net/aspose.pdf/document/printscaling/) property has been added to the [Document](https://reference.aspose.com/pdf/net/aspose.pdf/document/) class with values ​​Aspose.Pdf.PrintScaling.AppDefault or Aspose.Pdf.PrintScaling.None.
+The [PrintScaling](https://reference.aspose.com/pdf/net/aspose.pdf/document/printscaling/) property has been added to the [Document](https://reference.aspose.com/pdf/net/aspose.pdf/document/) class with values `​​Aspose.Pdf.PrintScaling.AppDefault` or `Aspose.Pdf.PrintScaling.None`.
+
+The page scaling option that shall be selected when a print dialog is displayed for this document. Valid values are `None`, which indicates no page scaling, and `AppDefault`, which indicates the conforming reader's default print scaling. If this entry has an unrecognized value, `AppDefault` should be used. Default value: `AppDefault`.
```

Comment:
The page scaling option shall be selected when a print dialog is displayed for this document

Reply:
Fixed

Reply:
It's from the PDF specification. This option controls the behavior of a print dialog, for example, in the Adobe Acrobat: if a dialog has an option "Print Scaling", what value will be selected there by default